src/VirtualClient/VirtualClient.Actions/HPLinpack/HPLinpackExecutor.cs

```diff
@@ -249,17 +269,61 @@ private void ThrowIfPlatformIsNotSupported()
                     $"'{PlatformSpecifics.LinuxX64}', '{PlatformSpecifics.LinuxArm64}'.",
                     ErrorReason.PlatformNotSupported);
             }
+            else if (this.Platform == PlatformID.Unix && this.CpuArchitecture != Architecture.Arm64 && this.UsePerformanceLibraries == true)
```

Comment:
See if you set the UsePerformanceLibrary to true by default. this profile is now default to crash on x64

Reply:
Agree. I have changed the default value in the profile to false and forgot to change in the executor. Thanks for pointing this out.